README.md

```diff
@@ -45,47 +45,44 @@ https://napari.org/plugins/index.html
 -->
 
 ## Installation using Command Line
-### 1. Prerequisites
-
-The plugin requires [Conda](https://docs.anaconda.com/anaconda/install/).
-- [Installing on Windows ](https://docs.anaconda.com/anaconda/install/windows/) 
-  - Follow the steps linked above except
-  - On step 8, check top the box to add to PATH
-  - ![Alt text](napari_allencell_annotator/assets/windowsstep8.png)
-- [Installing on Mac ](https://docs.anaconda.com/anaconda/install/mac-os/) 
-
-### 2. Install the plugin
-Click the link corresponding to your OS.
-#### [Windows](https://alleninstitute-my.sharepoint.com/:u:/g/personal/r_dhamrongsirivadh_alleninstitute_org/EexXIxeeIbNEs4KMjimcXOcBMn2J2QwxJhNEkOcRHC1eVg?e=JKa5WI)
-- From the link above, click the three dots on the top menu bar and select download. 
-- Open a file explorer and go to the Downloads folder. Use **Option 1** below. A prompt window should open and start installing. If this fails use **Option 2**. 
-  - **Option 1**: Double-click the file _install_napari.sh_
-  - **Option 2**: Search the file finder for Anaconda Prompt. Open version 3. Run the following commands one line at a time. 
+### 1. Install the plugin
+#### Windows
+- Open terminal
+- Create a virtual environment. We recommend using conda and python 3.10.
+  - To create a conda virtual environment, run the following commands one line at a time. 
     - conda create -n napari_annotator python=3.10 anaconda
     - conda activate napari_annotator
-    - python -m pip install --upgrade pip
+  - To create a venv virtual environment, run the following commands.
+    - python -m venv venv
+    - venv\Scripts\activate
+- Install Napari and the annotator plugin by running
     - python -m pip install "napari[all]"
     - python -m pip install napari-allencell-annotator
     - napari
```

Comment:
this is to start napari via command line, shouldnt be under install

Reply:
I added a step to open Napari and verify the plugin is installed in https://github.com/aics-int/napari-allencell-annotator/pull/108/commits/a4037bd8bb1b4a5964fc6825d20050f2c1a03d93